projects/igniteui-angular/src/lib/grids/api.service.ts

```diff
@@ -225,6 +227,7 @@ export class GridBaseAPIService <T extends IgxGridBaseDirective & GridType> {
         this._update_row(row, value);
 
         const args = row.createEditEventArgs();
+        args.owner = this.grid;
```

Comment:
Same applies here .. this should be moved inside createEditEventArgs() method of IgxRow class.